scripts/src/middleware.ts

```diff
@@ -83,7 +83,8 @@ export const reportMetrics = function(req: express.Request, res: express.Respons
 
 	res.on("finish", () => {
 		const path = req.route ? req.route.path : "unknown";
-		metrics.timeRequest(performance.now() - t, req.method, path);
+		const appId = req.context!.user!.appId;
```

Comment:
you won't always have the app_id here. Any request that isn't authorized doesn't have the context. req.context will be undefined and undefined.user will raise an exception